rust/janitor.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Rust",
+  "description": "Rust is a safe, concurrent, practical language.",
+  "icon": "https://janitor.technology/img/rust.svg",
+  "docker": {
+    "image": "janx/rust"
+  },
+  "ports": {
+    "22": {
+      "proxy": "none"
+    },
+    "8088": {
+      "label": "VNC",
+      "proxy": "https"
+    },
+    "8089": {
+      "label": "Cloud9",
+      "proxy": "https"
+    }
+  },
+  "scripts": {
+    "Build": "./x.py build",
+    "Build (stage1 quick)": "./x.py build --stage 1 src/libtest",
+    "Check": "./x.py check",
+    "Update source code": "git fetch origin && git rebase origin/master",
```

Comment:
Nit: This won't actually work, because `&&` isn't supported in Cloud9 IDE runners (they get passed as an argument...). Please use `git pull --rebase origin master` instead.